smolyay/benchmark/biggs.py

```diff
@@ -9,6 +9,14 @@ def domain(self):
                 [-1.96500000271271e-07, 17.99999982315],
                 [-5.0000001349, 13.49999987859]]
 
+    @property
+    def global_minimum(self):
+        return 0
+    
+    @property
+    def global_minimum_location(self):
+        return [0.9999999832, 9.9999998035, 4.9999998651]
```

Comment:
Same here. In general in these files, could you comment on where the really long decimals come from?

Reply:
According to @cakieslich, these were the best answers that the solvers found from wherever he got them. That may be the exact solution, or maybe the solver stopped short.

In that vein, I did some analysis to see if rounding decreased the error between the minimum recorded and one at the given location. Rounding to 0 decimal places improved the error for 31 problems, and of those 31, 24 problems got 0 error. Changing the number of decimal places would change the number of improved problems slightly, so i want to do some more investigating to find the best min location.

 I'll do a commit changing the ones that got 0 error and see how that changes things.

Reply:
There were 5 functions where the rounding brought them farther from the recorded locations than usual, an around 1e-3 difference. These are himmelbb sisser, powell, s256, and s275. Looking at the functions written down, though, it's clear all of them have a global minima at 0 for all variables.